src/NativeAdComponents.js

```diff
@@ -104,8 +104,8 @@ export const IconView = (props) => {
   if (!nativeAdView) return null;
 
   return (
-    nativeAd?.url ? <Image {...props} source={{uri: nativeAd.url}} /> :
-      nativeAd?.image ? <Image {...props} ref={imageRef} /> : 
```

Comment:
Good idea to remove the optional chaining operator `?`

Reply:
The default value of nativeAd is an empty map, so it should be good without the operator.